### PR TITLE
EDRD Phase-1 Development

### PR DESCRIPTION
**Pull request by deepakmulamalla**
EDRD Phase-1 Development Changes (Sprints-[1-5])

### CODE REVIEW

**Review by cgijeffolsen — APPROVED** [review]
General review completed. @ckleinacn - review still required.
cc @RanadheerRG @NataliaNikishina

**Review by RanadheerRG — APPROVED** [review]
@deepakmulamalla Review completed
Waiting for @ckleinacn Approval
cc: @cgijeffolsen  @NataliaNikishina

**Review by NataliaNikishina — APPROVED** [review]
@deepakmulamalla Composition not validated. Deployability appears to be positive. cc: @cgijeffolsen @RanadheerRG @ckleinacn

**Review by ckleinacn — APPROVED** [review]

**Pull request merged**
